staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -71,7 +79,7 @@ func NewCommandStartWardleServer(out, errOut io.Writer, stopCh <-chan struct{})
 	}
 
 	flags := cmd.Flags()
-	o.RecommendedOptions.AddFlags(flags)
+	o.addFlags(flags)
```

Comment:
optional nit: my personal preference is to keep the flag binding here.  I suspect we'll actually move admission into recommended options this release.  When we move it, I'd like to move the `AddFlags` back here.